Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -1337,7 +1358,7 @@ private extension ChatMessage {
             .sorted { $0.id.index < $1.id.index }
 
         latestReplies = {
-            guard !dto.replies.isEmpty else { return [] }
+            guard dto.replyCount > 0 else { return [] }
```

Comment:
We can also do the same optimization of the latest messages, with latestReplies:
```suggestion
            guard dto.replyCount > 0 else { return [] }
            return dto.replies.sorted.map()
```